catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/FileSystemPersistenceProvider.java

```diff
@@ -211,6 +212,20 @@ public void clear() {
     }
   }
 
+  public void storeJson(String key, Object value, Class<?> aClass) {
```

Comment:
❗️Instead of adding new Json specific method to this class, we should consider creating a common interface and have two implementations, one that uses Java object serialization and one that uses Json serialization.